src/enacrestic/__init__.py

```diff
@@ -10,6 +10,8 @@
 
 + ~/.enacrestic/bkp_include
   This is given to --files-from option
++ ~/.enacrestic/bkp_exclude
+  This is given to --exclulde-file option
```

Comment:
May I ask to fix the typo here ? `--exclulde-file`
thanks 😉

Reply:
Done